ExtractorUtils/HighAvailability/RawHighAvailabilityManager.cs

```diff
@@ -65,26 +69,14 @@ internal override async Task UpdateExtractorState()
                 var rows = await _destination.CogniteClient.Raw.ListRowsAsync<RawLogData>(_config.Raw?.DatabaseName, _config.Raw?.TableName).ConfigureAwait(false);
```

Comment:
Does this table store all states for all possible extractors?
If so is there a chance we will get too much data we don't need about all "other" extractors? Which seems to be not so good if there is a lot of data

Reply:
It's for High Availability, so an extractor needs to know about all instances of that same extractor. With correct configuration there should only be one small Raw row per standby extractor in that table, so a very small amount of data.